### PR TITLE
Dark Mode Buttons & Button Updates

### PR DESCRIPTION
**Pull request by megantrimble**

### CODE REVIEW

**Comment by jasoncypret** [comment]
I was expecting this to include the secondary button updates. Where are those updates?

**Comment by megantrimble** [comment]
Looks like that secondary button change didn't take the first time. It should be fixed now.

**Comment by megantrimble** [comment]
Playbook_buttons

File: app/pb_kits/playbook/pb_button/_button_mixins.scss [hunk 1]
```diff
@@ -114,3 +114,37 @@ $pb_button_border_width: 1px;
...
+    @include pb_button_hover($transparent);
+  }
+}
+
+// Dark Disabled ====================
+@mixin pb_button_disabled_dark {
+  $disabled_color: rgba($white, $opacity_5);
+  @include pb_button(rgba($slate, $opacity_5), $disabled_color);
+  pointer-events: none;
+}
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_button/_button_mixins.scss]: Seems nit-picky (and it is), but you will want to add a new line here

**Pull request merged**